Makefile

```diff
@@ -221,8 +221,10 @@ $(CLAPACK): $(CPYTHONLIB)
 	make -C CLAPACK
 
 
-build/packages.json: $(CPYTHONLIB) $(CLAPACK)
+build/packages.json: FORCE
 	make -C packages
 
 emsdk/emsdk/.complete:
 	make -C emsdk
+
+FORCE:
```

Comment:
If you don't add `FORCE` as a dependency, wouldn't `make -C packages` be run each time anyway?